bin/main

```diff
@@ -1,128 +1,79 @@
 #!/usr/bin/env ruby
 # rubocop:disable Metrics/BlockNesting
-# rubocop:disable Metrics/CyclomaticComplexity
-# rubocop:disable Metrics/PerceivedComplexity
-
-def board(sym1, sym2, arr1, arr2)
-  (1..9).each do |i|
-    if arr1.include?(i)
-      print " #{sym1} "
-    elsif arr2.include?(i)
-      print " #{sym2} "
-    else
-      print " #{i} "
-    end
-    print '|' unless (i % 3).zero?
-    puts "\n" if (i % 3).zero?
-  end
-end
-
-def validmove(move, arr1, arr2)
-  return false unless [1, 2, 3, 4, 5, 6, 7, 8, 9].include?(move)
-  return false unless !arr1.include?(move) && !arr2.include?(move)
-
-  true
-end
-
-def checkwin(move, arr)
-  if move != 6 && move != 8
-    return true if arr.include?(1) && arr.include?(4) && arr.include?(7)
-
-    return true if arr.include?(1) && arr.include?(5) && arr.include?(9)
-
-    return true if arr.include?(1) && arr.include?(2) && arr.include?(3)
-
-    return true if arr.include?(3) && arr.include?(5) && arr.include?(7)
-
-  end
-
-  if move != 2 && move != 4
-    return true if arr.include?(9) && arr.include?(8) && arr.include?(7)
-
-    return true if arr.include?(9) && arr.include?(6) && arr.include?(3)
-
-  end
-
-  if move != 1 && move != 3 && move != 7 && move != 9
-    return true if arr.include?(2) && arr.include?(8) && arr.include?(5)
-
-    return true if arr.include?(4) && arr.include?(6) && arr.include?(5)
-
-  end
-  false
-end
+require './lib/board_game.rb'
+require './lib/game_logic.rb'
+require './lib/player_info.rb'
```

Comment:
[Optional] But please fix it because on some devices code reviewers get errors due to this aspect which can result in requesting another code review.
- [ ] Using `require_relative` for files in your directory instead of `require` is a ruby convention.

  > "Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory."